lib/index.test.ts

```diff
@@ -587,7 +587,7 @@ describe('index.test.ts', () => {
 
   });
 
-  it('should be able to remove _id field', () => {
+  it('should be able to remove _id field by default', () => {
```

Comment:
removing the id by default would be changing the output for existing client code once the package is upgraded. for that reason, I would prefer the default to be false

Reply:
Sure, you're right